package.json

```diff
@@ -10,6 +10,11 @@
     "lint:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-brands-svg-icons": "^5.15.2",
+    "@fortawesome/free-regular-svg-icons": "^5.15.2",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/react-fontawesome": "^0.1.14",
```

Comment:
I see where we use `react-fontawesome`. Where do we use the rest of these dependencies?

Reply:
Confirmed that we do need these to render Font Awesome icons.